k8s/node-local-ha.yaml

```diff
@@ -56,7 +64,7 @@ data:
         forward . /etc/resolv.conf {
           force_tcp
         }
-        prometheus :9254
+        prometheus :9253
```

Comment:
wondering what we should use here https://github.com/prometheus/prometheus/wiki/Default-port-allocations